src/main/kotlin/com/jetbrains/snakecharm/string_language/SmkSLTokenTypes.kt

```diff
@@ -1,10 +1,38 @@
 package com.jetbrains.snakecharm.string_language
 
+import com.intellij.lang.*
+import com.intellij.psi.PsiElement
 import com.intellij.psi.tree.IElementType
+import com.intellij.psi.tree.ILazyParseableElementType
+import org.intellij.lang.regexp.RegExpLanguage
+import org.intellij.lang.regexp.RegExpParserDefinition
+
+class SmkSLTokenType(debugName: String) : IElementType(debugName, SmkSL)
+
+class RegExpElementType(debugName: String) : ILazyParseableElementType(debugName, SmkSL) {
+    override fun parseLight(chameleon: ASTNode): PsiBuilder {
+        val project = chameleon.psi.project
+        val regExpLanguage = RegExpLanguage.INSTANCE
+        val definition =
+                LanguageParserDefinitions.INSTANCE.forLanguage(regExpLanguage) as RegExpParserDefinition
+        val capabilities =
+                definition.defaultCapabilities
+        val lexer = definition.createLexer(project, capabilities)
```

Comment:
Not sure that default capabilities are ok for us, e.g. compare with PythonRegexpLanguage capabilities set + check runtime behavior.